datastore/migrations/migration_handler.py

```diff
@@ -255,7 +256,7 @@ def fill_id_sequences_table(self):
             max((split_part(fqid, '/', 2))::int) + 1 as id
             from models group by split_part(fqid, '/', 1)
```

Comment:
You should remove the slashes from the query. Shouldn't postgres complain here because the arg count doesn't match?

Reply:
arg, debugging.. :D